src/plugins/analysis/cve_lookup/test/test_db_setup.py

```diff
@@ -0,0 +1,67 @@
+from ..internal.database.db_setup import DbSetup
+from ..internal.helper_functions import CveEntry
+from ..internal.database.db_connection import DbConnection
+from ..internal.database.schema import Association, Cve, Cpe
+
+
+CVE_ENTRY = CveEntry(
+    cve_id='CVE-2023-1234',
+    impact={'cvssMetricV2': '5.0', 'cvssMetricV30': '6.0', 'cvssMetricV31': '7.0'},
+    summary='This is a test CVE',
+    cpe_entries=[
+        (
+            'cpe:2.3:o:vendor:product:version:update:edition:language:sw_edition:target_sw:target_hw:other',
```

Comment:
The string is hard to read and used in three places -> IMHO should be a constant

Reply:
I turned it into a constant.